tkg/client/upgrade_cluster_clusterclass.go

```diff
@@ -16,15 +16,20 @@ import (
 func (c *TkgClient) DoClassyClusterUpgrade(regionalClusterClient clusterclient.Client,
 	currentClusterClient clusterclient.Client, options *UpgradeClusterOptions) error {
 
+	if err := currentClusterClient.PatchClusterAPIAWSControllersToUseEC2Credentials(); err != nil {
```

Comment:
Should it be executed right after provider upgrade?

Reply:
Right after https://github.com/vmware-tanzu/tanzu-framework/blob/fe8dd986a4313e34518dcea3d7025c3b8c2e84e0/tkg/client/upgrade_region.go#L135